src/app/_components/NavBar.tsx

```diff
@@ -36,7 +36,7 @@ const Navbar: React.FC<Props> = ({ user }) => {
   return (
     <AppBar position="sticky">
       <Toolbar>
-        <img src={"../../public/logo.png"} width={50} height={50} alt="logo" />
+        <img src={"logo.png"} width={50} height={50} alt="logo" />
```

Comment:
What was the issue with the icon? This should also be a separate branch/ticket.

Reply:
For some reason, it was not loading in for me when I spun up the app. I fixed it by making the change and then it showed up for me.